.github/workflows/nightly-elixir.yml

```diff
@@ -51,13 +51,23 @@ jobs:
           mix run generate.exs elixir $SPEC_PATH $TEMPLATE_PATH $ELIXIR_OUTPUT_PATH
       - name: Commit files
         run: |
-          cd aws-beam/aws-elixir
-          git config --local user.email "aws-elixir@github.com"
-          git config --local user.name "AWS Elixir"
-          git add lib/aws/generated/*
-          if [ -n "$(git status --porcelain)" ]
+          cd aws/aws-sdk-go/
+          git fetch --tags
+          LATEST_TAG=$(git describe --tags --abbrev=0)
+          cd ../../
+          cd onno-vos-dev/aws-elixir
+          [[ -f latest-tag-aws-sdk-go ]] && LATEST_TAG_BUILT=$(<latest-tag-aws-sdk-go) || LATEST_TAG_BUILT=''
```

Comment:
This line is only needed for the first time essentially as the `latest-tag-aws-sdk-go` won't exist yet on `aws-elixir` and `aws-erlang` repo. Once merged and the files exist, I can follow up with a simplification :+1: